packages/browser-wallet/src/popup/pages/Account/Earn/Baking/utils.ts

```diff
@@ -38,7 +38,7 @@ function openStatusFromText(status: OpenStatusText): OpenStatus {
 }
 
 export const getExistingBakerValues = (accountInfo: AccountInfo): NotOptional<ConfigureBakerFlowState> | undefined => {
-    if (!isBakerAccountV1(accountInfo)) {
+    if (accountInfo.type !== AccountInfoType.Baker || accountInfo.accountBaker.version === 0) {
```

Comment:
We decided in the SDK to rely more on discriminative unions to identify types instead of functions checking properties of unknown objects. We needed these functions previously to be able to discriminate union types returned by the grpc v1 API.

If you look at the underlying types, there is no `AccountInfoBakerV1` and `AccountInfoBakerV0` anymore.